scripts/nayduck.py

```diff
@@ -203,6 +207,24 @@ def main():
         'tests': list(tests)
     }
 
+    if args.run_locally:
```

Comment:
This should go on line 195, before `code_path = …`. If we’re running locally there’s no need to have the auth code or get the current branch or sha. (This is actually similar to --dry-run added in the other commit which also could be done without getting the code and post data).

Reply:
@mina86 Yes, that's a good idea.